app/controllers/settings/general.js

```diff
@@ -24,10 +25,19 @@ export default Controller.extend(SettingsSaveMixin, {
     _scratchFacebook: null,
     _scratchTwitter: null,
 
+    imgMimeTypes: 'image/gif,image/jpg,image/jpeg,image/png,image/svg+xml',
```

Comment:
That's what I thought, too.. But the normal image upload didn't work anymore without setting them explicitly... Maybe there's another way to fix that?